### PR TITLE
chore: separating app UI configs slice

### PR DESCRIPTION
**Pull request by aforaleka**
separate ui configs from configs slice, and run migration to redux persist

testing:
- locally change theme to light on main branch
- switch to this branch and see if it's still light
- restart chrome and check if it's still light

### CODE REVIEW

File: src/state/migrations/__test__/3.test.ts [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+  },
+};
+
+const oppositeUiConfigsLocalStorageValues = {
+  appThemeSetting: AppTheme.Light,
+  appColorMode: AppColorMode.RedUp,
+  hasSeenLaunchIncentives: true,
+  defaultToAllMarketsInPositionsOrdersFills: false,
+  displayUnit: DisplayUnit.Fiat,
+  shouldHideLaunchableMarkets: true,
```
Comment (jaredvu) [inline src/state/migrations/__test__/3.test.ts:28]: default should be `false`

Comment (jaredvu) [reply src/state/migrations/__test__/3.test.ts:28]: unless this is part of the test

**Review by jaredvu — COMMENTED** [review]
Overall looks good to me! Will let @tinaszheng comment on `redux-persist`

File: src/state/migrations/__test__/3.test.ts (hunk 1 above)
Comment (aforaleka) [reply src/state/migrations/__test__/3.test.ts:28]: part of the test! this is the opposite values

**Review by tyleroooo — APPROVED** [review]

**Comment by tinaszheng** [comment]
🎉

**Pull request merged**